File: foundation/foundation 3 month/lesson8/homework/ex_11.py
def check(x):
	if x[1]=='IT':
		return x[0]


k=int(input("Xodimlar soni : "))
arr=dict()
for x in range(1,k+1):
	name=input(f"{x} chi xodim ismi: ")
	depa=input(f"{x} chi xodim bo'lim : ")
	arr[name]=depa
ms=tuple(arr.items())
print(ms)
ls=tuple(filter(check,ms))
print(ls)

File: full stack/full stack 1 month/lesson3/classwork/ex_11.js
function check(ms)
{
   let arr={}
    for(let i=0;i<ms.length;i++)
    {
        if (ms[i] in arr)
            {
                arr[ms[i]]+=1   
            }
        else
        {
                arr[ms[i]]=1
        }
    }
    return arr
}

let sonlar = [1, 2, 2, 3, 1, 4, 2, 1];
console.log(check(sonlar))